pegasus/migrations/115_add_contact_rollups_pardot_id_index.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:contact_rollups) do
+      add_index :pardot_id
```

Comment:
Should this index be unique? Presumably, we don't want to rows describing the same Pardot contact?